test/integration/client-side-encryption/client_side_encryption.prose.06.corpus.test.js

```diff
@@ -20,7 +20,9 @@ describe('Client Side Encryption Prose Corpus Test', function () {
 
   const corpusDir = path.resolve(__dirname, '../../spec/client-side-encryption/corpus');
   function loadCorpusData(filename) {
-    return EJSON.parse(fs.readFileSync(path.resolve(corpusDir, filename)), { relaxed: false });
+    return EJSON.parse(fs.readFileSync(path.resolve(corpusDir, filename)), {
+      relaxed: false
```

Comment:
mind reverting this? just cus its whitespace change, no need to modify history here (only asking since I'm asking for other changes)